db/db_sst_test.cc

```diff
@@ -1629,6 +1630,52 @@ TEST_F(DBSSTTest, GetTotalSstFilesSize) {
   ROCKSDB_NAMESPACE::SyncPoint::GetInstance()->DisableProcessing();
 }
 
+TEST_F(DBSSTTest, GetTotalSstAndBlobFilesSize) {
```

Comment:
We need to a better way to test the code...

Reply:
Right. We can create a database that has some blob files, close it, and then use a `SyncPoint` to make sure that `GetFileSize` does not get called when we reopen it.

Reply:
One way to do it would be to use `MockEnv` in the test (which has a `MockFileSystem`) and put the sync point in `MockFileSystem::GetFileSize`.